knimin/lib/data_access.py

```diff
@@ -2425,3 +2553,38 @@ def _revert_ready(self, barcodes):
                  SET results_ready = NULL
                  WHERE barcode IN %s"""
         self._con.execute(sql, [tuple(barcodes)])
+
+    def _add_new_samples(self, sample_ids):
+        """Test helper to create new samples to test with
+
+        Parameters
+        ----------
+        sample_ids : list of str
+            Sample ids to create
+        """
+        sql = """INSERT INTO pm.sample (sample_id) VALUES (%s)"""
+        self._con.executemany(sql, [[x] for x in sample_ids])
+
+    def _delete_new_samples(self, sample_ids):
+        """Test helper to delete the samples that were just created
+
+        Parameters
+        ----------
+        sample_ids : list of str
+            Sample ids to delete
+        """
+        sql = """DELETE FROM pm.sample WHERE sample_id = %s"""
```

Comment:
should this `DELETE` be restricted to a particular plate? For instance, what about the scenario where the same sample is run multiple times? One risk is that, right now this method is used as a test helper, but it may get reused inadvertently in the future.